bench/FileSizes.hs

```diff
@@ -6,7 +6,9 @@ module FileSizes where
 import Control.Exception
 import Control.Monad.IO.Class
 import Data.IORef
+#ifdef VERSION_polysemy
 import Data.Tuple
```

Comment:
AFAIK code for `eff` also uses this. You can remove this and add `eff_swap` and `poly_swap` functions below.

Reply:
Done.